Sources/JWTKit/EdDSA/EdDSA.swift

```diff
@@ -13,23 +13,19 @@ public extension EdDSA {
     /// A struct representing a public key used in EdDSA (Edwards-curve Digital Signature Algorithm).
     ///
     /// In JWT, EdDSA public keys are represented as a single x-coordinate and are used for verifying signatures.
-    /// Currently, only the ``Curve/ed25519`` curve is supported.
+    /// Currently, only the ``EdDSACurve/ed25519`` curve is supported.
     struct PublicKey: EdDSAKey {
-        // https://www.rfc-editor.org/rfc/rfc8037.html#appendix-A.2
```

Comment:
Out of interest, should this be public?

Reply:
The struct is in a public extension so it is effectively public if not specified otherwise (otherwise the tests wouldn't pass)

Reply:
I do not enjoy this implicit access control in Swift 🙃